controllers/controller-games.js

```diff
@@ -1,109 +1,144 @@
-        .then((review) => {
-            if (!review) {
-                return Promise.reject({
-                    status: 404,
-                    msg: `This review doesn't exist!`,
-                });
-            } else {
-                res.status(200).send({ review })
-            }
-        })
-        .catch(next);
-}
+  updateReviewByID(review_id, inc_votes)
+    .then((review) => {
+      if (!review) {
+        return Promise.reject({
+          status: 404,
+          msg: `This review doesn't exist!`,
+        });
+      } else {
+        res.status(200).send({ review });
+      }
+    })
+    .catch(next);
+};
 
 exports.getAllUsers = (req, res, next) => {
-    selectUsers()
-        .then((users) => {
-            res.status(200).send({ users });
-        })
-        .catch(next);
+  selectUsers()
+    .then((users) => {
+      res.status(200).send({ users });
+    })
+    .catch(next);
 };
 
 exports.getAllReviews = (req, res, next) => {
-    selectReviews()
-        .then((reviews) => {
-            res.status(200).send({ reviews });
-        })
-        .catch(next);
-}
+  selectReviews()
+    .then((reviews) => {
+      res.status(200).send({ reviews });
+    })
+    .catch(next);
+};
 
 exports.getCommentsByID = (req, res, next) => {
-    const { review_id } = req.params;
+  const { review_id } = req.params;
 
-    if (isNaN(review_id)) {
-        return next({
-            status: 400,
-            msg: `'${review_id}' is not a valid review number!`,
+  if (isNaN(review_id)) {
+    return next({
+      status: 400,
+      msg: `'${review_id}' is not a valid review number!`,
+    });
+  }
+  selectReviewByID(review_id)
+    .then((review) => {
+      if (!review) {
+        return Promise.reject({
+          status: 404,
+          msg: "There are no comments because this review doesn't exist!",
         });
-    }
-    selectReviewByID(review_id)
-        .then((review) => {
-            if (!review) {
-                return Promise.reject({
-                    status: 404,
-                    msg: "There are no comments because this review doesn't exist!",
-                });
-            } else {
-                return selectCommentsByID(review_id)
-            }
-
-        })
-        .then((comments) => {
-                res.status(200).send({ comments });
-        })
-        .catch(next);
+      } else {
+        return selectCommentsByID(review_id);
+      }
+    })
+    .then((comments) => {
+      res.status(200).send({ comments });
+    })
+    .catch(next);
+};
 
+exports.createComment = (req, res, next) => {
+  const { review_id } = req.params;
+  if (typeof req.body.username !== "string" || typeof req.body.body !== "string") {
+    return next({
+        status: 400,
+        msg: `Body doesn't contain mandatory keys`,
+      });
+  }
 
-}
\ No newline at end of file
+  addComment(req.body, review_id)
+    .then((newComment) => {
+      res.status(201).send({ newComment });
```

Comment:
I'd recommend returning this on the key of 'comment'

Reply:
Okay, changed as well!